frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import {Button} from '$lib/components/button';
+  export let password = '';
+  export let autoComplete = '';
+  let passwordRevealed = false;
+  /** variable used to refer to the input box in code to change its type*/
+  let inputbox: HTMLInputElement;
+  /** function that hides and reveals the password and changes the icon of the button*/
+  const changeRevealed = () => {
+    passwordRevealed = !passwordRevealed;
+    if (passwordRevealed) {
+      inputbox.type = 'text';
+    } else {
+      inputbox.type = 'password';
+    }
+  };
+</script>
+
+<!--
+@component
+PasswordField is an input box for password that comes with a button
+to reveal and hide the password
+
+### Properties
+
+- `autoComplete` : variable used for field's password auto-fill value
```

Comment:
The docs says auto-fill but it is actually setting the value for autocomplete

Reply:
I suppose it would also be logical to rename the autoComplete prop to be all lower-case, i.e., `autocomplete`